week-8/Homework/mandatory/1-practice/1-practice.md

```diff
@@ -26,6 +26,12 @@ The following endpoint is publicly available from Github
 
 <!-- Write your answer here -->
 
+{owner} - github username
+{repo} - repository name
+{pull_number} - the pull request number
+
 2. Describe in a sentence what this API endpoint returns when all of the fields are completed?
 
 <!-- Write your answer here -->
+
+it should show the comments on the specified pull request.
```

Comment:
More precisely it takes you to the comment tab of the PR page.
Under the hood, it redirects you to the PR comment page. If you use `curl` command (not sure if it is available on windows by default) you can see what you get exactly. (It can be seen in the browser console/network tab but preserve log needs to be turned on otherwise network list might be cleared when a redirect occurs.)
```bash
curl https://github.com/CodeYourFuture/js-exercises/pull/993/comments
# Result:
<html><body>You are being <a href="https://github.com/CodeYourFuture/js-exercises/pull/993">redirected</a>.</body></html>%
```

Reply:
I didn't try to access the URL, I only compared it to one of my PR's URL 😄